mappings/net/minecraft/fluid/Fluid.mapping

```diff
@@ -27,10 +27,10 @@ CLASS net/minecraft/class_3611 net/minecraft/fluid/Fluid
 		ARG 1 fluid
 	METHOD method_15781 setDefaultState (Lnet/minecraft/class_3610;)V
 		ARG 1 state
-	METHOD method_15782 getVelocity (Lnet/minecraft/class_1922;Lnet/minecraft/class_2338;Lnet/minecraft/class_3610;)Lnet/minecraft/class_243;
+	METHOD method_15782 getFlowDirectionVector (Lnet/minecraft/class_1922;Lnet/minecraft/class_2338;Lnet/minecraft/class_3610;)Lnet/minecraft/class_243;
```

Comment:
`FluidState#getVelocity` should also be renamed for consistency.

Reply:
Done. I have changed it there so that every method that overrides or implements will get the Javadocs. https://github.com/FabricMC/yarn/pull/3284/commits/5076644a8c3933dec14c80c9f6b0da20b352fffc